dotCMS/src/main/java/com/dotcms/contenttype/business/RelationshipFactoryImpl.java

```diff
@@ -552,6 +554,25 @@ public void addRelationship(String parent,String child, String relationType)thro
             TreeFactory.saveTree(tree);
         }
     }
-    
 
+    @Override
+    public List<Relationship> getOneSidedRelationships(final ContentType contentType, int limit, final int offset) throws DotDataException {
+        DotConnect dc = new DotConnect();
+        StringBuilder sql = new StringBuilder();
+
+        if (limit == 0)
+            throw new DotDataException("limit param must be more than 0");
```

Comment:
Could we print more information?, for example ct name, etc.

Reply:
Maybe use DotPreconditions.checkArgument(limit>0). This throws IllegalArumentException

Reply:
@oarrietadotcms creo que no hace falta ya que es solo validando el limit. El ct name sería irrelevante